examples/python_bert_squad_example/bert-squad-migraphx.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import json
+import time
+import os.path
+from os import path
+import sys
+
+import tokenization
+from run_onnx_squad import *
+
+print(sys.path)
+
+migx_lib_path = "/opt/rocm/lib"
+if migx_lib_path not in sys.path:
+    sys.path.append(migx_lib_path)
```

Comment:
The MIGraphX package should setup the python path when installed, so this seems unnecessary. Perhaps we have a bug in our packaging.

Reply:
We use these site packages for python [here](https://github.com/RadeonOpenCompute/rocm-cmake/blob/master/share/rocm/cmake/ROCMCreatePackage.cmake#L138). Perhaps, that is not the correct path on newer pythons.

Reply:
I see in the python notebook it shows the path `/usr/local/lib/python3.6/dist-packages` which we don't set up. Let me update this in the rocm-cmake.

I think these lines should be removed, and we fix our packages.

Reply:
Makes sense, thank you Paul. I will remove the lines.

Reply:
Tested .deb you proposed, it works, removed the workaround. Thanks!